crates/ruff_linter/resources/test/fixtures/pyupgrade/UP040.py

```diff
@@ -46,3 +46,7 @@ class Foo:
 # OK
 x: TypeAlias
 x: int = 1
+
+# UP040 won't raise if a generic is repeated.
```

Comment:
It looks like we _are_ actually emitting a violation here, right? (And I think that's correct!)

```suggestion
# Make sure "T" appears only once
# in the type parameters for the modernized type alias
```

Reply:
Yes, thank you!